### ISSUE TITLE
Can't export when defining extra pandoc arguments containing spaces

### ISSUE THREAD

**Issue by mgmeyers** [OliverBalfour/obsidian-pandoc#77]
I use a reference document with pandoc like so:

```
--reference-doc=/Users/matt/Library/Mobile Documents/iCloud~md~obsidian/Documents/Knowledge Garden/Templates/essay-template.docx
```

But because "Extra Pandoc arguments" are split on spaces, the plugin sees this as three arguments rather than one. 

https://github.com/OliverBalfour/obsidian-pandoc/blob/master/pandoc.ts#L141

This throws the error:

```
pandoc: Documents/iCloud~md~obsidian/Documents/Knowledge: openBinaryFile: does not exist (No such file or directory)
```

**Comment by Limezy** [comment]
Hi, 
Have you tried using "\" as an escape ?

```
--reference-doc=/Users/matt/Library/Mobile Documents/iCloud~md~obsidian/Documents/Knowledge\ Garden/Templates/essay-template.docx
```

**Comment by mgmeyers** [comment]
@Limezy Yeah, but no luck. The value of "Extra Pandoc arguments" is split on space characters before it's sent to the command line, so escaping that space has no effect.

**Comment by ndegroot** [comment]
I can confirm this. I did find a [workaround for (path) spaces in plugin settings](https://github.com/OliverBalfour/obsidian-pandoc/issues/81#issuecomment-1013507755) Please test it.

**Comment by naive231** [comment]
> workaround for (path) spaces in plugin settings

I think you're missing another '\\'. It should be placed in front of "Documents" of "Mobile Documents" like:

`--reference-doc=/Users/matt/Library/Mobile\ Documents/iCloud~md~obsidian/Documents/Knowledge\ Garden/Templates/essay-template.docx`
